src/lib/seam/components/SupportedDeviceTable/SupportedDeviceBrandSection.tsx

```diff
@@ -0,0 +1,90 @@
+import classNames from 'classnames'
+import type { DeviceModel } from 'seamapi'
+
+import { getBrandInfo, getImage } from 'lib/brands.js'
+import { ChevronRightIcon } from 'lib/icons/ChevronRight.js'
+import { HiddenDevicesOverlay } from 'lib/seam/components/SupportedDeviceTable/HiddenDevicesOverlay.js'
+import { ShowAllDevicesButton } from 'lib/seam/components/SupportedDeviceTable/ShowAllDevicesButton.js'
+import { SupportedDeviceRow } from 'lib/seam/components/SupportedDeviceTable/SupportedDeviceRow.js'
+import { useToggle } from 'lib/ui/use-toggle.js'
+
+/**
+ * How many device models before requiring the
+ * user to expand the list.
+ */
+const numDevicesBeforeHiding = 3
```

Comment:
```suggestion
const maxDevicesBeforeCollapsing = 3
```

Reply:
Updated in https://github.com/seamapi/react/pull/251/commits/031af09b71ff6c2d62011973ff98c287eddfdcb9